cmd/network-wait-launcher/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"time"
+)
+
+func main() {
+	// example: /kpack/network-wait-launcher gcr.io -- completion --notary-url="some-url"
+
+	hostname := os.Args[1]
+	command := os.Args[3:]
+
+	if err := waitForDns(hostname); err != nil {
+		log.Fatal(err)
+	}
+
+	cmd := exec.Command(command[0], command[1:]...)
```

Comment:
Should we consider using `syscall.Exec` here? this would completely replace the network-wait-launcher Go process with the launcher process instead of spawning a new one. 

1. Maybe more closely mirrors linux buildpods
2. Are there edge cases around using subprocesses here?
  - Exit codes
  - Process interrupts
  - Error messages
3. Does `syscall.Exec` work on windows?

Reply:
it is [not supported](https://github.com/golang/go/issues/30662) on windows.. but maybe worth thinking about.

Reply:
We should still think through exit codes and process interrupts.

Reply:
I'm going to put this in a separate issue